### PR TITLE
Teste lol =D

### PR DESCRIPTION
**Pull request by dr1nk0rdi3**
Wtf

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1 +1 @@
-print('Hello, BIAAATCH!')
\ No newline at end of file
+print('wtf')
```
Comment (rzro) [inline main.py]: WTF?!

Comment (dr1nk0rdi3) [inline main.py]: yes!! WTF

Comment (rzro) [inline main.py]: # DUDE
Aight, man. *Cool*, _cool_! Geez 👍  🥇

**Comment by rzro** [comment]
Done. Closed!

**Pull request merged**